<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultMenu</class>
 <widget class="QDialog" name="ResultMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#nextButton:hover{background-color: rgba(0,0,0,0.5);}
QPushButton#nextButton{background-color:rgba(0,0,0,0); border-radius:15px;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#restartButton:hover{background-color: rgba(0,0,0,0.5);}
QPushButton#restartButton{background-color:rgba(0,0,0,0); border-radius:15px;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#menuButton:hover{background-color: rgba(0,0,0,0.5);}
QPushButton#menuButton{background-color:rgba(0,0,0,0); border-radius:15px;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetHallOfFame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
